e2e/security/security.go

```diff
@@ -63,27 +62,34 @@ func (c ctx) testSecurityUnpriv(t *testing.T) {
 		// capabilities
 		{
 			name:       "capabilities_keep_true",
-			argv:       []string{"ping", "-c", "1", "8.8.8.8"},
+			argv:       []string{"grep", "^CapEff:", "/proc/self/status"},
 			opts:       []string{"--keep-privs"},
-			expectExit: 255,
+			expectExit: 255, // singularity errors out, --keep-privs needs root
 		},
 		{
-			name:       "capabilities_keep-false",
-			argv:       []string{"ping", "-c", "1", "8.8.8.8"},
-			expectExit: 2,
+			// we start without any capabilities, the
+			// expected set is empty.
+			name:     "capabilities_keep-false",
```

Comment:
I think this needs to be an underscore between `keep` and `false`.

Reply:
believe me, I held back touching that sort of thing. I would have changed so much :-\